### PR TITLE
Layout tweaks

### PR DESCRIPTION
**Pull request by humdingerb**

### CODE REVIEW

**Comment by humdingerb** [comment]
These deal with the transfer and split/transaction windows. It among other things uses a bit more spacing.

* Transfer
Before:
transfer_old

After:
transfer_new

* Split/Transaction:
Before:
transaction_old

After:
transaction_new

File: src/SplitView.cpp [hunk 1]
```diff
@@ -137,42 +160,56 @@ SplitView::SplitView(const char* name, const TransactionData& trans, const int32
...
+		.Add(new BSeparatorView(B_HORIZONTAL, B_PLAIN_BORDER))
+		.AddStrut(B_USE_SMALL_SPACING)
 		.AddGrid(1.0f, 0.0f)
-		.Add(fAddSplit, 0, 0)
-		.Add(fRemoveSplit, 1, 0)
-		.AddGlue(2, 0)
-		.Add(fSplitTotal, 3, 0)
-		.AddGlue(4, 0)
-		.End()
+			.Add(fAddSplit, 0, 0)
```
Comment (korli) [inline src/SplitView.cpp:166]: if you want to keep this formatting, you'll have to add `// clang-format off  ` then `// clang-format on`

File: src/TransferWindow.cpp [hunk 2]
```diff
@@ -126,22 +119,33 @@ TransferWindow::InitObject(Account* src, Account* dest, const Fixed& amount)
...
-		.Add(fAmount, 2, 2, 2)
-		.AddGrid(1.0f, 1.0f, 2, 3, 2)
-		.AddGlue(0, 0)
-		.Add(fCancel, 1, 0)
-		.Add(fOK, 2, 0)
-		.End()
-		.End()
+	BLayoutBuilder::Group<>(this, B_VERTICAL, B_USE_HALF_ITEM_SPACING)
+		.SetInsets(B_USE_WINDOW_SPACING)
+		.AddGroup(B_HORIZONTAL)
```
Comment (korli) [inline src/TransferWindow.cpp:124]: ditto

File: src/SplitView.cpp (hunk 1 above)
Comment (humdingerb) [reply src/SplitView.cpp:167]: Thanks for the reminder! PR was updated accordingly.

**Comment by korli** [comment]
LGTM

**Comment by humdingerb** [comment]
Let's merge then!

**Pull request merged**